app/_includes/md/plugins-hub/kong-process-errors.md

```diff
@@ -1,9 +1,12 @@
 <!---shared with logging plugins --->
 
-This logging plugin logs HTTP request and response data, and also supports streams
+This logging plugin logs HTTP request and response data, and also supports stream
 data (TCP, TLS, and UDP).
 
-If you are looking for the Kong process error file (which is the nginx error file),
+If you are looking for the Kong process error file (which is the Nginx error file),
```

Comment:
```suggestion
The Kong process error file is synonymous with the Nginx error file and you can find it at the following path:
```

Reply:
Not the best, but I tried to reword this a bit.

Reply:
I think it's even more straightforward than that: Kong uses the Nginx error file as its process error file.
Our own config ref says this about log level settings:

> Log level of the Nginx server. Logs are found at `<prefix>/logs/error.log`.
> See http://nginx.org/en/docs/ngx_core_module.html#error_log for a list of
accepted values.